03_letters/madhavansu/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	m := make(map[rune]int)
```

Comment:
You can have cleaner initialisation code with `m := map[rune]int{}`. This is more of a preference thing.

Reply:
Updated.